Packs/Absolute/Integrations/Absolute/Absolute.py

```diff
@@ -919,6 +1125,16 @@ def main() -> None:  # pragma: no cover
             secret_key=secret_key,
             x_abs_date=x_abs_date,
         )
+
+        client_v3 = ClientV3(
+            base_url=base_url,
+            verify=verify_certificate,
+            proxy=proxy,
+            headers=HEADERS_V3,
```

Comment:
We can change the init method of the Client to get the HEADERS_V3 if no headers were passed... and here not to pass headers.